src/anndata/extensions.py

```
@@ -0,0 +1,110 @@
+    "register_formatter",
+    "TypeFormatter",
+    "SectionFormatter",
+    "FormattedOutput",
+    "FormattedEntry",
+    "FormatterContext",
+    "FormatterRegistry",
+    "formatter_registry",
+    # Type hint utilities
+    "extract_uns_type_hint",
+    "UNS_TYPE_HINT_KEY",
+]
```

Ruff (0.14.8)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)